test/lib/linter/UI5YamlLinter.ts

```diff
@@ -0,0 +1,52 @@
+import test from "ava";
+import UI5YamlLinter from "../../../src/linter/yaml/UI5YamlLinter.js";
+import LinterContext from "../../../src/linter/LinterContext.js";
+
+test("Test UI5YamlLinter report (parsing and analyzing)", async (t) => {
+	/* Mock resource content of ui5.yaml file,
+	(formatted as used in src/detectors/typeChecker/index.ts - #analyzeFiles()),
+	(contains relevant 'framework' property and 'libraries' sub-property),
+	(contains only deprecated libraries) */
+	const resourceContent =
+`specVersion: '3.0'
+metadata:
+  name: ava-test-ui5yamllinter
+type: application
+framework:
+  name: OpenUI5
+  version: "1.121.0"
+  libraries:
+    - name: sap.ca.scfld.md
+    - name: sap.ca.ui
+    - name: sap.fe.common`;
+
+	const resourcePath = "/ui5.js"; // '.js' due to renaming in src/detectors/typeChecker/index.ts - #analyzeFiles()
```

Comment:
is this renaming still necessary after the recent refactoring? If it is, let's sync with @RandomByte on how to solve this

Reply:
03af6ea ?

Reply:
LGTM